framework/resource/logresource/resource_test.go

```diff
@@ -331,3 +320,4 @@ func (r *testResource) ApplyUpdateChange(ctx context.Context, obj, updateState i
 func (r *testResource) Underlying() framework.Resource {
 	return r
 }
+*/
```

Comment:
I just commented the tests to verify the real behaviour within an operator. I will see what I can do towards testing in case we want to go with this approach I suggest here.